unitex/src/StaticValue.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+//package model;
+
+import java.util.ArrayList;
+
+import fr.umlv.unitex.config.Config;
+
+/**
+ *
+ * @author Rojo Rabelisoa
+ */
+public class StaticValue {
```

Comment:
`StaticValue` doesn't seems to be the most  evocative name for this class... Do you have any other suggestion?

Reply:
DictionnaryPath?